media/js/firefox/os/firefox-os-2.5.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+;(function($) {
+    'use strict';
+
+    $('.cta-button').on('click', function(e) {
+        e.preventDefault();
```

Comment:
If we're adding a tracking pixel to this page, we should respect Do Not Track like we do elsewhere. We have a [handy global helper](https://github.com/mozilla/bedrock/blob/master/media/js/base/dnt-helper.js) that makes this easy in just a few lines of code. Something like:

    if (!window._dntEnabled()) {
        e.preventDefault();
        ftGoalTag14004();
    }